httpapi/coreapi/coreapi.go

```diff
@@ -28,14 +28,14 @@ func Register(r *chi.Mux,
 		tokenRegistry: nftSrv.(documents.TokenRegistry),
 	}
 	r.Post("/documents", h.CreateDocument)
```

Comment:
The router automatically adds the prefix `v1` to all paths that it handles right? Otherwise we would have to add the `/v1` prefix to the paths here, right?

Reply:
it adds automatically